core/project_settings.cpp

```diff
@@ -572,7 +572,7 @@ Error ProjectSettings::_load_settings_text(const String p_path) {
 				if (section == String()) {
 					set(assign, value);
 				} else {
-					set(section + "/" + assign, value);
+					set(section.plus_file(assign), value);
```

Comment:
Why it being a property name prevents from using `plus_file()`. I kinda understand but property name still can't have `//` in them so I don't really see the problem.

Reply:
Sorry for butting in, but because it's not a file name, I would argue that you shouldn't use functions meant for filenames here, as that can actually make it harder to read, not easier.

Reply:
Exactly, we never know if `plus_file` could be changed in the future to use a backslash `\` on Windows, for example.

Reply:
I understand your points but lets no kid ourselves, It's not like `plus_file` is going to change anywhere soon (probably not once in the lifetime of our universe).

Reply:
No, of course not, but it still makes it harder to read what the intended effect is supposed to be.